src/org/isf/patient/service/PatientIoOperationRepository.java

```diff
@@ -21,6 +21,8 @@
 
     @Query(value = "SELECT * FROM PATIENT WHERE (PAT_DELETED='N' OR PAT_DELETED IS NULL) ORDER BY PAT_NAME, ?#{#pageable}", nativeQuery= true)
     public List<Patient> findAllWhereDeleted(Pageable pageable);
+
+    public List<Patient> findAllByDeletedIsNullOrDeletedEqualsOrderByName(String patDeleted, Pageable pageable);
```

Comment:
Why the @Query annotation is missing here?

Reply:
Because this is not a custom Query but the ORM deriving the query from the method name directly.

Reply:
I see, thanks!